oshi-core/src/main/java/oshi/driver/linux/proc/LinuxOSThread.java

```diff
@@ -0,0 +1,140 @@
+/**
+ * MIT License
+ *
+ * Copyright (c) 2010 - 2020 The OSHI Project Contributors: https://github.com/oshi/oshi/graphs/contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package oshi.driver.linux.proc;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.InvalidPathException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import oshi.software.common.AbstractOSThread;
+import oshi.util.FileUtil;
+import oshi.util.ParseUtil;
+import oshi.util.platform.linux.ProcPath;
+
+public class LinuxOSThread extends AbstractOSThread {
+
+    private static final Logger LOG = LoggerFactory.getLogger(LinuxOSThread.class);
+
+    private int processID;
+    private int threadID;
+    private String name;
+    private String exePath;
+    private String commandLine;
+    private String userID;
+    private String user;
+    private String groupID;
+    private String group;
+
+
+    public LinuxOSThread(int pID, int tID) {
+        this.processID = pID;
```

Comment:
Personal preference, for short names like this should be all lowercase (`pid`, `tid`). Above I think I'd prefer `processId` to `processID` although I've done both elsewhere in the code...

Reply:
Ok. I think it is fine if we use either format, but consistency matter. Hence i will use `processId` and similar variable names elsewhere.